models/engine/db_storage.py

```diff
@@ -74,3 +74,16 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ Retrieves an object from the database. """
+        if cls in classes.keys() or cls in classes.values():
+            if isinstance(cls, str):
+                cls = eval(cls)
```

Comment:
I believe its best we stick to a class instance as **cls** or better still create a class method to parse this across the db_model as using either string or class instance is a bit inconsistent